src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { ReactNode, useState, useEffect } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useColonyDomainsQuery, AnyToken, OneDomain } from '~data/index';
+import { Address } from '~types/index';
+import ColorTag from '~core/ColorTag';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+import Numeral from '~core/Numeral';
+import { ColonyActionTypes } from '../types';
```

Comment:
Please sort the imports list, with a space between them:
- `node_module` imports at the top
- "our" modules
- types/utils
- styles

It's the same thing I've mentioned here: https://github.com/JoinColony/colonyDapp/pull/2312#discussion_r538238038